core/model/modx/modaccesspermission.class.php

```diff
@@ -1,7 +1,13 @@
 <?php
-/**
- * @package modx
+/*
+ * This file is part of MODX Revolution.
+ *
+ * Copyright (c) MODX, LLC. All Rights Reserved.
+ *
+ * For complete copyright and license information, see the COPYRIGHT and LICENSE
+* files found in the top-level directory of this distribution.
```

Comment:
Just nitpicking: Why is that line not indented?
A lot of files below seems to have this issue.

Reply:
There were a lot of regex global search and replaces going on. I could have messed up a few. Will search for and resolve that.